File: src/Vincent OS/UAC.vb
Public Class UAC
    ' (!) Vérification du mot de passe (!)
    Private Sub verifylog()
        If TextBox1.Text = "Vincent OS" Then
            TextBox1.Text = ""
            '' /!\ Faire attention aux potentiels changements dans le futur ! /!\
            If Applications.Visible = True Then
                Store.Show()
                Me.Close()
            ElseIf Bureau2.Panel1.Visible = True Then
                Paramètres.Show()
                Me.Close()
            ElseIf Bureau2.Button11.Visible = True Then
                Store.Show()
                Me.Close()
            End If
        Else
            Dim backmusic As New System.Media.SoundPlayer(My.Resources.errorSound)
            backmusic.Play()
            MessageBox.Show("ERREUR : Le mot de passe est incorrecte.", "ERREUR", MessageBoxButtons.OK, MessageBoxIcon.Error)
        End If
    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        verifylog()
    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        Me.Close()
    End Sub

    Private Sub TextBox1_KeyPress(ByVal sender As System.Object, ByVal e As System.Windows.Forms.KeyPressEventArgs) Handles TextBox1.KeyPress
        If e.KeyChar = Convert.ToChar(Keys.Enter) Then
            verifylog()
        End If
    End Sub

    Private Sub UAC_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        My.Computer.Audio.PlaySystemSound(System.Media.SystemSounds.Exclamation)
    End Sub
End Class

File: src/Vincent OS/UAC.cs
using System;
using System.Windows.Forms;

namespace Vincent_OS
{
    public partial class UAC
    {
        public UAC()
        {
            InitializeComponent();
        }
        // /!\ Vérification du mot de passe /!\
        private void verifylog()
        {
            Bureau2 bureau2 = new Bureau2();
            if (TextBox1.Text == "Vincent OS")
            {
                TextBox1.Text = "";
                // /!\ Faire attention aux potentiels changements dans le futur ! /!\
                if (bureau2.Panel1.Visible == true)
                {
                    Paramètres paramètres = new Paramètres();
                    paramètres.Show();
                    Close();
                }
            }
            else
            {
                var backmusic = new System.Media.SoundPlayer(My.Resources.Resources.errorSound);
                backmusic.Play();
                MessageBox.Show("ERREUR : Le mot de passe est incorrecte.", "ERREUR", MessageBoxButtons.OK, MessageBoxIcon.Error);
            }
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            verifylog();
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            Close();
        }

        private void TextBox1_KeyPress(object sender, KeyPressEventArgs e)
        {
            if (e.KeyChar == Convert.ToChar((int)Keys.Enter))
            {
                verifylog();
            }
        }

        private void UAC_Load(object sender, EventArgs e)
        {
            System.Media.SystemSounds.Exclamation.Play();
        }
    }
}